scipy/signal/fir_filter_design.py

```diff
@@ -126,6 +158,73 @@ def kaiserord(ripple, width):
     ----------
     Oppenheim, Schafer, "Discrete-Time Signal Processing", p.475-476.
 
+    Examples
+    --------
+    We will use the Kaiser window method to design a lowpass FIR filter
+    for a signal that is sampled at 1000 Hz.
+
+    We want at least 65 dB rejection in the stop band, and in the pass
+    band the gain should vary no more than 0.5%.
+
+    We want a cutoff frequency of 175 Hz, with a transition between the
+    pass band and the stop band of 24 Hz.  That is, in the band [0, 163],
+    the gain varies no more than 0.5%, and in the band [187, 500], the
+    signal is attenuated by at least 65 dB.
+
+    >>> from scipy.signal import kaiserord, firwin
+    >>> import matplotlib.pyplot as plt
+    >>> fs = 1000.0
+    >>> cutoff = 175
+    >>> width = 24
+
+    The Kaiser method accepts just a single parameter to control the pass
+    band ripple and the stop band rejection, so we use the more restrictive
+    of the two.  In this case, the pass band ripple is 0.005, or 46.02 dB,
+    so we will use 65 dB as the design parameter.
+
+    Use `kaiserord` to determine the length of the filter and the
+    parameter for the Kaiser window.
+
+    >>> numtaps, beta = kaiserord(65, width/(0.5*fs))
+    >>> numtaps
+    167
+    >>> beta
+    6.20426
+
+    Use `firwin` to create the FIR filter.
+
+    >>> taps = firwin(numtaps, cutoff, window=('kaiser', beta),
+    ...               scale=False, nyq=0.5*fs)
+
+    Use the discrete Fourier transform to compute the frequency
+    response of the filter.  ``w`` is the array of frequencies,
+    and ``h`` is the corresponding complex array of frequency responses.
+
+    >>> h = np.fft.rfft(taps, 8192)
+    >>> w = np.fft.rfftfreq(8192, 1/fs)
```

Comment:
Yeah we can make `freqz` use FFT (and RFFT) when `N` (int) is given instead of `w` (array), right? I'm surprised that it doesn't already exist.

I might have time to make a PR if you for that if you want one fewer thing on your todo list :)